proposals/0065-collections-move-indices.md

```diff
@@ -472,7 +472,7 @@ in the interest of full disclosure:
   manipulations end up looking like free function calls:
 
   ```swift
-  let j = index(after: i)           // self.successor(i)
+  let j = index(after: i)           // self.index(after: i)
```

Comment:
I am not entirely sure about this change, but I think the existing comment makes no sense.